src/CopyToastmasterDetailsToRoles.js

```diff
@@ -185,12 +185,26 @@ function resetToastmasterDetailsFormulas() {
   // NOTE(acmiyaguchi): variables are generally defined in
   // CopySignUpSheetToRoles, but I don't appreciate the design choice of gs to
   // share variable scope across files. It's very confusing.
-  currentDateCell.setFormula(`='${SIGNUP_SHEET_NAME}'!C${SIGNUP_START_ROW}`);
+  // Also, it may be useful to make this an integer offset from the start of the
```

Comment:
Good call, a function to map from int to a letter of the alphabet would make sense.

Reply:
I'll file an issue for this, since it's not a super trivial function (e.g. after `Z` comes `AA` and so forth).

Reply:
Thanks!